src/ray/protobuf/experimental/autoscaler.proto

```diff
@@ -150,7 +145,11 @@ message GetClusterResourceStateReply {
   // Cluster resource constraints.
   // There could be multiple constraints issued by different
   // jobs. Autoscaler to make sure all constraints are satisfied.
-  repeated ClusterResourceConstraint cluster_resource_constraints = 6;
+  // QQ:
+  // 1. I don't think we support job-level constraints currently,
+  // 2. It will be a cluster level constrain. There is only one constraint.
+  // 3. The constraint is currently overridden by each sdk.request_resources()
+  optional ClusterResourceConstraint cluster_resource_constraints = 6;
```

Comment:
^ cc @scv119

Reply:
i think it'd be nice to keep it as a per job constraints, since each job might not have global information of the resources required by other jobs..

cc @raulchen @jjyao what do you think?

Reply:
I think the first step is to match the existing behavior, but leave the api flexible so we can move the system towards the desired direction.

Reply:
Yea, I agree. Per job is nice but we need to think about how autoscaler should aggregate them.